client/src/components/ProductOverview/ProductOverviewAPIUtils.js

```diff
@@ -5,7 +5,8 @@ let url = 'http://localhost:3000';
 export default {
```

Comment:
axios already returns a promise, so you shouldn't have to wrap it in one, just 'return axios.get(....)'